### PR TITLE
Adapt modal and inlinevalidation to work with barceloneta LTS

### PR DESCRIPTION
**Pull request by petschki**
Due to Bootstrap 4 markup changes in plone.app.z3cform widgets the patterns

- pat-plone-modal
- pat-inlinevalidation

need some markup change.

### CODE REVIEW

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Comment by thet** [comment]
If the changes are not backwards compatible with Plone 5.2, then we need to keep the barceloneta-LTS changes for mockup and plone.staticresources in seperate branches and include the specific branch in package.json in plone.staticresources. Or are they backwards compatible?

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Comment by petschki** [comment]
@thet changes are all backwards compatible ... I've started Jenkins once more ...

**Comment by petschki** [comment]
@thet related mockup PR: https://github.com/plone/mockup/pull/975

File: news/76.feature [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Adapte pat-plone-modal and pat-inlinevalidation to work with barceloneta LTS
```
Comment (thet) [inline news/76.feature:1]:
```suggestion
Adapt ``pat-plone-modal`` and ``pat-inlinevalidation`` to work with barceloneta LTS.
Add missing ``plone.svg`` icon.
```

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**